feed/views.py

```diff
@@ -1,3 +1,100 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http.response   import JsonResponse
+from django.views           import View
+
+from .models                import Feed, Comment, ImageUrl
+from user.models            import User
+from product.models         import Product
+
+class FeedEntireView(View):    
+    def get(self, request):
+        MAXIMUM_COMMENT = 2
+        feed_list   = []
+        start_pos   = int(request.GET.get('offset'))
+        div         = int(request.GET.get('limit'))
+    
+        for feed in Feed.objects.all().order_by('-id'):
+
+            # 피드 정보
+            feed_basic_data = {
+                'feed_id'               : feed.id, 
+                'feed_user'             : feed.user.nickname,
+                'created_at'            : feed.created_at,
+                'description'           : feed.description,
+                'like_number'           : feed.like_number,
+                'tag_item_number'       : feed.tag_item_number,
+                'feed_main_image'       : feed.imageurl_set.values('image_url').first()
+                }
+
+            # 피드의 상품 정보
+            if feed.product_id:
+                product_feed_id         = feed.product
+                product_id              = feed.product.id 
+                product_name            = feed.product.name
+                discount_rate           = feed.product.discount_rate
+                price                   = feed.product.price
+                product_image           = Product.objects.get(id=product_id).productimageurl_set.get(is_main=1).image_url
+
+                product_data = {
+                    'product_name'      : product_name,
+                    'discount_rate'     : discount_rate,
+                    'price'             : price,
+                    'product_image'     : product_image
+                    }
+            else:
+                product_data = {
+                    'product_name'      : '',
+                    'discount_rate'     : '',
+                    'price'             : '',
+                    'product_image'     : ''
+                }
+            
+            # 피드의 댓글
+            if feed.comment_set.all():
```

Comment:
```suggestion
            if feed.comment_set.exists():
```
있는지 없는지를 확인할 때에는 `all()`보다는 `exists()`를 활용하시는 것이 좋습니다 👍